### PR TITLE
Change-Projectile cleanup (and now you can be ANY* species)

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
:cl: Cobby
experiment: Being hit by a bolt of change now opens up previously-barred races. Hopefully you don't roll plasmaman!
/:cl:
* If the subtype for the species was added hehe

I was going to brag about how there wasn't any balance change but this does allow for more of the "fun" races within humans to come up.

Webbit so probably some dumb typo somewhere

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
ok ill just wait until I have access to github/dm ffs

File: code/modules/projectiles/projectile/magic.dm [hunk 1]
```diff
@@ -152,111 +147,65 @@
...
+			if(M.ckey)
+				Xe = pick(/mob/living/carbon/alien/humanoid/hunter,/mob/living/carbon/alien/humanoid/sentinel)
 			else
-				if(!M.ckey)
-					new_mob = new /mob/living/simple_animal/hostile/alien/sentinel(M.loc)
-				else
-					new_mob = new /mob/living/carbon/alien/humanoid/sentinel(M.loc)
-
+				Xe = pick(/mob/living/carbon/alien/humanoid/hunter,/mob/living/simple_animal/hostile/alien/sentinel)
+			new Xe(M.loc)
```
Comment (XDTM) [inline code/modules/projectiles/projectile/magic.dm]: new mob =?

File: code/modules/projectiles/projectile/magic.dm [hunk 2]
```diff
@@ -131,18 +131,13 @@
...
-					new_mob = new path(M.loc)
-				if("syndiborg")
-					path = pick(typesof(/mob/living/silicon/robot/modules/syndicate))
-					new_mob = new path(M.loc)
-				if("drone")
-					new_mob = new /mob/living/simple_animal/drone/polymorphed(M.loc)
+			var/robot = pick(200;/mob/living/silicon/robot,
+							/mob/living/silicon/robot/modules/syndicate,
+							/mob/living/silicon/robot/modules/syndicate/medical,
+							200;/mob/living/simple_animal/drone/polymorphed)
```
Comment (Cyberboss) [inline code/modules/projectiles/projectile/magic.dm]: Um...

File: code/modules/projectiles/projectile/magic.dm (hunk 1 above)
Comment (Cyberboss) [reply code/modules/projectiles/projectile/magic.dm]: what @XDTM said

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
Changelog

File: code/modules/projectiles/projectile/magic.dm [hunk 3]
```diff
@@ -152,111 +147,65 @@
...
+							/mob/living/simple_animal/pet/cat/cak,
+							/mob/living/simple_animal/chick)
 			new_mob = new path(M.loc)
 
 		if("humanoid")
-			new_mob = new /mob/living/carbon/human(M.loc)
+			if(prob(50))
+				new_mob = new /mob/living/carbon/human(M.loc)
+			else
+				var/hooman = pick(subtypesof(/mob/living/carbon/human/species))
```
Comment (lzimann) [inline code/modules/projectiles/projectile/magic.dm]: not how species work

Comment (ExcessiveUseOfCobblestone) [reply code/modules/projectiles/projectile/magic.dm]: The species subtype does all the hard work for us.

File: code/modules/projectiles/projectile/magic.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/projectiles/projectile/magic.dm]: pickweight is default 100 so 200 is twice as much.

It's equivalent to as it was previously.

File: code/modules/projectiles/projectile/magic.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/projectiles/projectile/magic.dm]: good catch

File: code/modules/projectiles/projectile/magic.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/projectiles/projectile/magic.dm]: http://www.byond.com/docs/guide/chap16.html#16.1.4

File: code/modules/projectiles/projectile/magic.dm (hunk 3 above)
Comment (XDTM) [reply code/modules/projectiles/projectile/magic.dm]: god i can't wait to remove human species subtypes, they clog the spawn list when i'm testing so i keep having to scroll down to spawn a human

Comment (ExcessiveUseOfCobblestone) [reply code/modules/projectiles/projectile/magic.dm]: too late now sucker hehe >:)

**Pull request merged**